src/components/Faq.astro

```diff
@@ -15,7 +15,7 @@ import faq from '../assets/data/faq.json';
           Can't find the answer you're looking for? You can always <a
             href="mailto:team@hackhpi.org"
             class="font-medium text-hackhpi-purple">send us an email</a
-          > with your enquiry.
+          > with your enquiry. Furthermore, you can find the MLH's code of conduct <a href="https://static.mlh.io/docs/mlh-code-of-conduct.pdf">here</a>.
```

Comment:
```suggestion
          > with your enquiry. Furthermore, you can find the MLH's code of conduct
          <a
            href="https://static.mlh.io/docs/mlh-code-of-conduct.pdf"
            target="_blank"
            rel="noopener noreferrer"
            class="font-medium text-hackhpi-purple">here</a
          >.
```

Reply:
Sure, I haven't used Astro before, and kept it at basic HTML, so this is your part, I'm not qualified to evaluate this.